iOS/Managers/AchievementsManager.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  AchievementsManager.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 3/16/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import ParseLiveQuery
+
+class AchievementsManager {
+    
+    enum LocalAchievementType: String {
```

Comment:
Added this as a helper for achievements that are triggered locally because now all achievement types are queried .